### ISSUE TITLE
New release tag 

### ISSUE THREAD

**Issue by trungn1** [hyperopt/hyperopt#930]
Hi! it looks like it's been a long time since the last tag, Nov 2021, could we get a more official tag published please? Thank you.

**Comment by celestinoxp** [comment]
I agree! This package has the latest version in 2021... we are in 2024!!! Please post the latest code release on pypi.

@bluestealth @jaberg @yamins81 @ExpectationMax @dandxy89 @temporaer @npinto @akx @albertz

**Comment by nicwest** [comment]
is there any update on this?

**Comment by celestinoxp** [comment]
ping @FrankwaP
